### PR TITLE
Jetpack Settings: Disable Related Posts sub-settings instead of hiding

### PR DESCRIPTION
**Pull request by tyxla**
This PR updates the sub-settings of the Related Posts card in General Settings to get disabled instead of being hidden when the corresponding module is not active. As suggested by @MichaelArestad and @rickybanister.

#### To test

* Checkout this branch.
* Go to `/settings/general/$site` where `$site` is one of your sites.
* Verify settings are not hidden, but are disabled when the module is disabled.
* Verify settings are working as before when module is enabled.

### CODE REVIEW

**Comment by oskosk** [comment]
LGTM

**Comment by rickybanister** [comment]
@MichaelArestad let's discuss what to do with the **Preview** when the setting is toggled off. I think it's confusing to see the feature turned off, but the preview appear 'active'.

It's likely a good thing to show an 'example' of what Related Posts does, but I want to avoid any confusion where a user might think that the related posts will appear on their site regardless of the setting.

I actually think we've introduced a regression here—I swear that in the past, before we started on settings work, the radio button interface did indeed 'dim' the preview when the feature was turned off. 

So to recap, should we 'dim' the Preview when the feature is off? Should we change the 'Preview' label to 'Example' instead to be more specific? 'Preview' is inaccurate if the feature is toggled off.

**Comment by MichaelArestad** [comment]
@rickybanister I think we can add some text explaining how cool it is and switch the label to "Example." I'd rather not dim it. I don't think it ever dimmed in Jetpack.

**Comment by tyxla** [comment]
> I actually think we've introduced a regression here—I swear that in the past, before we started on settings work, the radio button interface did indeed 'dim' the preview when the feature was turned off.

@rickybanister if you recall, we removed the preview dimming in favor of totally hiding it in #10423, as suggested by you in #10391. I've removed the dimming there, because it didn't make sense to have it when preview was being hidden, and it was also using a deprecated CSS class. I'm happy to bring this back if you folks decide it's better for users.

> I think we can add some text explaining how cool it is and switch the label to "Example." 

I've altered the label to "Example" as suggested.

Folks, feel free to mark the PR as "Ready to Merge" once design is approved.

**Comment by rickybanister** [comment]
@MichaelArestad can you work up a version of the text you mention above? 

I think it would be fine if we switch to 'Example' and dim it when the parent toggle is off. No text necessary. 

What about parity with the 'view in customizer' text in Jetpack? That sentence/link is not represented here, do we want it to be?

**Comment by MichaelArestad** [comment]
@rickybanister I'm okay without a longer description as well.

**Comment by rickybanister** [comment]
> What about parity with the 'view in customizer' text in Jetpack? That sentence/link is not represented here, do we want it to be?

I confused myself and forgot that Related Posts is only available in the customizer on the remote side so disregard that comment.

**Comment by rickybanister** [comment]
🚢
